resources/styles/components/reference-book/navbar.less

```diff
@@ -15,9 +24,26 @@
       }
     }
     .menu-toggle:before{ content: @fa-var-list; }
+    // don't collapse navbar when screen is narrow
+    @media (max-width: 979px) {
```

Comment:
Maybe  That's overriding the bootstrap default.   I'll investigate to see if bootstrap has a bar for it. If not I'll make a comment to that effect